src/BuildingRegistry.Api.BackOffice/BuildingUnit/BuildingUnitController-NotRealize.cs

```diff
@@ -67,10 +67,7 @@ public partial class BuildingUnitController
             }
             catch (AggregateNotFoundException)
             {
-                throw CreateValidationException(
-                    ValidationErrorCodes.Building.BuildingNotFound,
-                    string.Empty,
-                    ValidationErrorMessages.Building.BuildingNotFound);
+                throw new ApiException(ValidationErrorMessages.BuildingUnit.BuildingNotFound, StatusCodes.Status404NotFound);
```

Comment:
Should not be 404. The resource is `.../gebouweenheden/x/...` so if `x` is not found, it can be a 404. However we're not looking at `x` here, but it's parent. So just a 400 (previous version)